### PR TITLE
EPMRPP-50872 Add code reference

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: modules/cucumber-reportportal-formatter.js [hunk 1]
```diff
@@ -3,6 +3,12 @@ const ReportPortalClient = require('reportportal-client');
 const Path = require('path');
 const pjson = require('../package.json');
 
+const formatCodeRef = (path, itemName) => {
+  const codeRef = path.replace(/\\/g, '/');
```
Comment (KatsiarynaTatarynovich) [inline modules/cucumber-reportportal-formatter.js:7]: Will this separator `/\\/g` be in other systems, in Linux, for example?

Comment (AmsterGet) [reply modules/cucumber-reportportal-formatter.js:7]: As I know `/` used in Unix systems, so we just need to replace `\`.

Comment (ElenaRomanchuk) [reply modules/cucumber-reportportal-formatter.js:7]: It won't be. Separator in POSIX system is `/`, as required. So this regex just for Windows `/` separator.

Comment (AmsterGet) [reply modules/cucumber-reportportal-formatter.js:7]: http://www.mit.edu/~6.005/fa09/resources/avoid-dependent-code.html

**Pull request merged**